packages/repository-json-schema/src/build-schema.ts

```diff
@@ -271,15 +271,54 @@ export function getNavigationalPropertyForRelation(
   }
 }
 
+/**
+ * Checks the options and generates a descriptive suffix using compatible chars
+ * @param options json schema options
+ */
 function getTitleSuffix<T extends object>(options: JsonSchemaOptions<T> = {}) {
+  const sep = {
+    start: '_',
+    end: '_',
+    delimiter: '-',
+  };
+
+  return getTextSuffix(options, sep);
+}
+
+/**
+ * Checks the options and generates a descriptive suffix
+ * @param options json schema options
+ */
+function getDescriptionSuffix<T extends object>(
+  options: JsonSchemaOptions<T> = {}
+) {
+  const sep = {
+    start: '[',
+    end: ']',
+    delimiter: ',',
+  };
+  const suffix = getTextSuffix(options, sep);
+
+  return suffix !== '' ?  `(Model options: ${suffix})` : '';
```

Comment:
1. Let's use `Schema options` (or `Model schema options`) instead of `Model options` please. The options are changing the schema generated from the model, the model itself remains unchanged.

2. I am not sure if the complexity of the current implementation is worth the benefits. Have you considered using JSON instead of a custom text format?

    ```ts
    return options ? JSON.stringify(options, null, 2) : undefined;
   ```

    Alternatively, if we want something slightly prettier than JSON, then let's use [stringify-object](https://www.npmjs.com/package/stringify-object), we have had a good experience with that package in `lb4` CLI.

    https://github.com/strongloop/loopback-next/blob/c944654881140119e44fe9e1db85a0f26e59d658/packages/cli/lib/utils.js#L539-L548

Reply:
1. Done
2. I took that approach to keep readability reusing logic applied to title suffix. To clarify, you suggest putting the stringified version of options as description suffix?

Reply:
> To clarify, you suggest putting the stringified version of options as description suffix?

Yes, exactly.

Reply:
We created the format `Optional[name]Excluding[id]` as a middle ground between what OpenAPI/JSON Schema supports and what is still readable by humans.

Since the description field is a free-form text, there is no longer need to limit ourselves to a compressed title, and we can use `JSON.stringify` or `stringify-object` for the best readability.

Reply:
changes applied (stringify-object)